rasa_core/remote.py

```diff
@@ -137,12 +137,16 @@ def upload_model(self, model_dir, max_retries=1):
         while max_retries > 0:
             max_retries -= 1
 
-            with io.open(model_zip, "rb") as f:
-                response = requests.post(url, files={"model": f})
-
-            if response.status_code == 200:
-                logger.debug("Finished uploading")
-                return response.json()
+            try:
+                with io.open(model_zip, "rb") as f:
+                    response = requests.post(url, files={"model": f})
+
+                if response.status_code == 200:
+                    logger.debug("Finished uploading")
+                    return response.json()
+            except Exception as e:
```

Comment:
I guess should be fine to just catch `RequestException`, right?

Reply:
I think it would be `ConnectionError` if anything?

Reply:
although that covers it

Reply:
don't care to much, but would avoid to catch all exceptions.